examples/sdk/train_api.py

```diff
@@ -0,0 +1,74 @@
+from kubeflow.training.api.training_client import TrainingClient
+from kubeflow.storage_init_container.hugging_face import (
+    HuggingFaceModelParams,
+    HuggingFaceTrainParams,
+    HfDatasetParams,
+    TRANSFORMER_TYPES,
+)
+from kubeflow.storage_init_container.s3 import S3DatasetParams
+from peft import LoraConfig
+from transformers import TrainingArguments
+import json
+
+client = TrainingClient(
+    config_file="/Users/deepanker/Downloads/deepanker-test-kubectl.cfg"
+)
+
+client.train(
+    name="deepanker-test",
+    namespace="test",
+    num_workers=2,
+    num_procs_per_worker=0,
+    storage_config={
+        "size": "10Gi",
+        "storage_class": "deepanker-test",
+    },
+    model_provider_parameters=HuggingFaceModelParams(
+        model_uri="hf://Jedalc/codeparrot-gp2-finetune",
+        transformer_type=TRANSFORMER_TYPES.AutoModelForCausalLM,
+    ),
+    dataset_provider_parameters=HfDatasetParams(
+        repo_id="imdatta0/ultrachat_10k",
+        access_token="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",
```

Comment:
I have deleted this access token, as it would be visible in previous commits

Reply:
I think that we shouldn't have such a token.
Should we take a token via an environment variable?

Reply:
+1 
Either change to public dataset or get access token as an env variable